events/KWoC/student-manual.md

```diff
@@ -0,0 +1,180 @@
+7. [Contact](#contact)
+
+## Introduction
+
+**Goals of the Program**: 
+
+- Kharagpur Winter of Code is an initiative by the Kharagpur Open Source Society (KOSS)
+  for students, who have little to no experience in open-source software development, to
+  get them involved during the winter vacations and break the ice of open-source 
+  contributions under the guidance of mentors who have the necessary experience.
+- KWoC is modeled to be an event similar to Google Summer of Code/Outreachy/Season of KDE/
+  Rails Girls Summer of Code/GirlScript Summer of Code and many similar programs. KWoC 
+  similarily, is aimed at introducing new students to the area and helping them make PRs.
+  In fact, a first for many of them, and also gaining confidence which will help them continue 
+  their open-source journey.
+
+Registered mentors can add their open-source projects to the list. Every project will have
+a dedicated place for ideas, which can either be on a README.md file or as an issue. This 
+page will contain the possible work to be done on the project. Students can then contact
+the mentor who shall guide them on the project and continue discussing their plan for the 
+winter.
+
+**What is Kharagpur Winter of Code?**
+
+The primary goals of KWoC are
+
+- Inspire young developers to get involved in open-source software development
+- Help them master the development workflow of Git and Github.
+- Connect students with mentors.
+
+## Why Should You Apply?
+
+You now know that participating in Kharagpur Winter of Code seems like a productive thing to do. 
+However, you may be wondering why you shouldn't just go on a trip instead. Here are some reasons
+why applying for KWoC is a good idea. 
+
+**An absolutely amazing working experience on projects**
+
+KWoC is a place where you get to use your current skill set on existing projects. Working
+in collaboration with other people on an open-source project is probably something new to you.
+
+**Acquiring new skill sets**
+
+While you'll be adding your piece of code to the existing software, you will get to learn
+its structure and more about the process of creating it from scratch. This will help you in creating your own software in the future.
+
+**Building your network**
+
+While being a part of KWoC, you will have healthy conversations with many helpful and talented mentors, This will help you build your network 
+in your early days. You can then approach them any time after the program ends. They remain with you, in 
+your connections.
+
+**Sense of Achievement**
+
+It feels really great to have your first patch submitted. It feels like an achievement when your contribution goes online and
+is ready to be helpful to other folks throughout the world.
+
+**For the love of code and open-source**
+
+If you love to code and it relates even distrinctly to your passion, then you will find an opportunity to express that love by being
+a part of Kharagpur Winter of Code.
+
+## Am I Good Enough?
```

Comment:
We should mention somewhere here that this is open for all students and not just KGPians.